modules/router-store/src/router_store_module.ts

```diff
@@ -96,7 +96,7 @@ export type RouterReducerState<T = RouterStateSnapshot> = {
 };
 
 export function routerReducer<T = RouterStateSnapshot>(
-  state: RouterReducerState<T>,
+  state: RouterReducerState<T> | undefined,
```

Comment:
Had to use `undefined` because the signature for an `ActionReducer`  is `(state: T | undefined, action: V)`